Exiled.Events/Patches/Events/Scp3114/Slapped.cs

```diff
@@ -0,0 +1,53 @@
+// -----------------------------------------------------------------------
+// <copyright file="Slapped.cs" company="Exiled Team">
+// Copyright (c) Exiled Team. All rights reserved.
+// Licensed under the CC BY-SA 3.0 license.
+// </copyright>
+// -----------------------------------------------------------------------
+
+namespace Exiled.Events.Patches.Events.Scp3114
+{
+    using System.Collections.Generic;
+    using System.Reflection.Emit;
+    using Exiled.API.Features.Pools;
+    using Exiled.Events.Attributes;
+    using Exiled.Events.Handlers;
+
+    using HarmonyLib;
+    using PlayerRoles.PlayableScps.Scp3114;
+    using Exiled.Events.EventArgs.Scp3114;
+    using static HarmonyLib.AccessTools;
+
+    /// <summary>
+    /// Patches <see cref="Scp3114Slap.DamagePlayers" />.
+    /// Adds the <see cref="Handlers.Scp3114.Slapped" /> event.
+    /// </summary>
+    [EventPatch(typeof(Scp3114), nameof(Scp3114.Slapped))]
+    [HarmonyPatch(typeof(Scp3114Slap), nameof(Scp3114Slap.DamagePlayers))]
+    internal class Slapped
+    {
+        private static IEnumerable<CodeInstruction> Transpiler(IEnumerable<CodeInstruction> instructions, ILGenerator generator)
+        {
+            List<CodeInstruction> newInstructions = ListPool<CodeInstruction>.Pool.Get(instructions);
+
+            newInstructions.InsertRange(0, new CodeInstruction[]
```

Comment:
You call your Event Slapp**ed** but it's call before the DamagePlayer method is executed and so we cannot differentiate a player who have get it's damage or not

Reply:
also it's do not contain a lot of information